src/tokenserver/db/models.rs

```diff
@@ -1080,7 +1080,7 @@ mod tests {
             service_id,
             email: email1.to_owned(),
             generation: 1,
-            client_state: "616161".to_owned(),
```

Comment:
The unit and integration tests previously set the client state equal to "aaa" (which is not valid hex), even though the client state is stored in the database as hex. I updated the client state values throughout the unit and integration tests to improve the consistency and make things easier to understand.